### ISSUE TITLE
In the dynamic BERTopic, do probability distributions of document-topic change over the time? 

### ISSUE THREAD

**Issue by ShuaiGuo95** [MaartenGr/BERTopic#850]
In the dynamic BERTopic, do probability distributions of document-topic change over the time? 
- If Yes, how to obtain the changable probability distributions?
- If No, are the probability distributions obtained by `topic_model.fit_transform()`?

**Comment by MaartenGr** [comment]
In dynamic BERTopic, we are not calculating probability distributions but the topic representations over time. The resulting `c-TF-IDF` matrices get updated depending on the split that is being made. 

> If No, are the probability distributions obtained by topic_model.fit_transform()?

Yes, if you have set `calculate_probabilities=True` then it will generate a topic-document probability matrix. Do note though that this does not take the potentially dynamic nature into account. You will have to create the mapping yourselves.

**Comment by ShuaiGuo95** [comment]
Thank you very much for your patience and reply!

Does that means, in BERTopic, topic-document probability matrix remains unchanged over the time?
If so, I think I can obtain a probability distribution with the topic-document probability matrix, with a step of normalization.
And I changed the source code to output words and importance of words of the topics over the time.

**Comment by ShuaiGuo95** [comment]
Thank you very much!
